src/test/scala/chiselTests/util/experimental/TruthTableSpec.scala

```diff
@@ -104,4 +104,13 @@ class TruthTableSpec extends AnyFlatSpec {
     assert(t.toString contains "111->?")
     assert(t.toString contains "     0")
   }
+
+  "Using TruthTable.fromEspressoOutput" should "merge rows on conflict" in {
+    val mapping = List(
+      (BitPat("b000?????1100011"), BitPat("b0100110")),
+      (BitPat("b001?????1100011"), BitPat("b0101110")),
+      (BitPat("b001?????1100011"), BitPat("b0100111"))
+    )
+    assert(TruthTable.fromEspressoOutput(mapping, BitPat("b???????????")).toString contains "0101111")
```

Comment:
Can you make this test actually check the expected mapping that comes out rather than just matching part of a `BitPat`? It should combine all 3 terms into 1 right?

Reply:
Yeah, I think as it's implemented it merges the last two but I think I should just get all 3 to merge

Reply:
Oh sorry, I missed that the first one had a different input pattern, yes please show it merging the last 2. I just want the test to show exactly what happens, like don't cast to a `String`, just show the expected simplified `TruthTable`.